src/terra-dev-site/doc/how-to.2/use-translated-strings.doc.mdx

```diff
@@ -0,0 +1,25 @@
+# How To Use Translated Strings
+
+Translated strings can be retrieved using the `intl` object provided by `injectIntl` HOC-generator provided by [react-intl](https://github.com/formatjs/react-intl/blob/master/docs/API.md#injectintl).
+
+```jsx
+import React from 'react';
+import { injectIntl, intlShape } from 'react-intl';
+
+const ExampleComponent = injectIntl(({ intl }) => {
```

Comment:
nit, can we follow this syntax to mach what we typically do:
```
const ExampleComponent = ({ intl }) =>
...
export default injectIntl(ExampleComponent)
```

Reply:
Honestly, I'd rather promote this pattern.